tests/session/session-advanceOperationTime-003.phpt

```diff
@@ -21,7 +21,7 @@ class MyTimestamp implements MongoDB\BSON\TimestampInterface
         return 1234;
     }
 
-    public function __toString()
+    public function __toString(): string
```

Comment:
This is an example of the BC break where users will need to add a type hint to any implementations of our BSON interfaces.